tests/driver_ws281x/Makefile

```diff
@@ -0,0 +1,13 @@
+BOARD ?= atmega328p
+include ../Makefile.tests_common
+
+# Update this to your needs
+PIN ?= GPIO_PIN(0, 0)
```

Comment:
Just choose a pin you like so it's easy to test without changing any code.

Reply:
Last time I did so Murdock was complaining. (`GPIO_PIN(0, 0)` seems to work on every board with GPIO_SUPPORT.)